### PR TITLE
[OPS-8259] build OPS and HID branches too

### PR DESCRIPTION
**Pull request by teodorescuserban**

### CODE REVIEW

**Review by lazysoundsystem — APPROVED** [review]
Okay with me

**Review by rupl — APPROVED** [review]

**Comment by rupl** [comment]
Or wait a minute. This is the SNAP repo but the branch names are for HID.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ on:
       - dev
       - main
       - master
+      - OPS-*
+      - ops-*
+      - HID-*
+      - hid-*
```
Comment (rupl) [inline .github/workflows/build.yml:12]:
```suggestion
      - OPS-*
      - ops-*
      - SNAP-*
      - snap-*
```

Comment (teodorescuserban) [reply .github/workflows/build.yml]: Yeah whatever, copy pasta failure. :)

**Review by cafuego — APPROVED** [review]
I guess? I'd just add `feature/**` and name my branches feature/OPS-foobar-wibble-blah or whatever.

**Comment by rupl** [comment]
I'm morally opposed to slashes in branch names. Yes, that's my (slanted) line.

**Pull request merged**